tutor/templates/apps/openedx/config/lms.env.json

```diff
@@ -27,6 +27,7 @@
   "CMS_BASE": "{{ CMS_HOST }}",
   "LMS_BASE": "{{ LMS_HOST }}",
   "CONTACT_EMAIL": "{{ CONTACT_EMAIL }}",
+  "CONTACT_MAILING_ADDRESS": "{{ PLATFORM_NAME }}@{{ LMS_HOST }}",
```

Comment:
Is this a good value? Can I suggest instead `{{ PLATFORM_NAME }} - {% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}`?

Reply:
My understanding is that the mailing address is meant to be an email address, is that incorrect?
![image](https://user-images.githubusercontent.com/13796098/124363312-62438600-dc32-11eb-9928-094a08673c13.png)

Reply:
No, I believe that the `CONTACT_MAILING_ADDRESS` is the physical mailing address, as indicated by the variable usage in the edx-platform code base:

```
$ git grep CONTACT_MAILING_ADDRESS
cms/envs/common.py:CONTACT_MAILING_ADDRESS = _('Your Contact Mailing Address Here')
common/djangoapps/student/email_helpers.py:            settings.CONTACT_MAILING_ADDRESS
lms/djangoapps/instructor/enrollment.py:        settings.CONTACT_MAILING_ADDRESS
lms/envs/common.py:CONTACT_MAILING_ADDRESS = 'SET-ME-PLEASE'
openedx/core/djangoapps/ace_common/template_context.py:            'CONTACT_MAILING_ADDRESS', site=site, site_config_name='contact_mailing_address'),
openedx/core/djangoapps/schedules/docs/README.rst:    CONTACT_MAILING_ADDRESS = '<insert_physical_address_here>'
openedx/core/djangoapps/schedules/tests/test_resolvers.py:    @override_settings(CONTACT_MAILING_ADDRESS='123 Sesame Street')
openedx/core/djangoapps/schedules/tests/test_resolvers.py:    @override_settings(CONTACT_MAILING_ADDRESS='123 Sesame Street')
```

Reply:
Ahh yeah makes sense!